orca/3.7-predicting-house-prices.ipynb

```diff
@@ -0,0 +1,1239 @@
+      "\u001b[2m\u001b[36m(pid=1366)\u001b[0m To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert\n",
+      "\u001b[2m\u001b[36m(pid=1366)\u001b[0m WARNING:tensorflow:From /home/zhenhao/.local/lib/python3.7/site-packages/tensorflow/python/data/ops/multi_device_iterator_ops.py:601: get_next_as_optional (from tensorflow.python.data.ops.iterator_ops) is deprecated and will be removed in a future version.\n",
+      "\u001b[2m\u001b[36m(pid=1366)\u001b[0m Instructions for updating:\n",
+      "\u001b[2m\u001b[36m(pid=1366)\u001b[0m Use `tf.data.Iterator.get_next_as_optional()` instead.\n",
+      "\u001b[2m\u001b[36m(pid=1366)\u001b[0m WARNING:tensorflow:AutoGraph could not transform <function preprocess at 0x7f1abc4c9dd0> and will run it as-is.\n",
+      "\u001b[2m\u001b[36m(pid=1366)\u001b[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.\n",
+      "\u001b[2m\u001b[36m(pid=1366)\u001b[0m Cause: Unable to locate the source code of <function preprocess at 0x7f1abc4c9dd0>. Note that functions defined in certain environments, like the interactive Python shell do not expose their source code. If that is the case, you should to define them in a .py source file. If you are certain the code is graph-compatible, wrap the call using @tf.autograph.do_not_convert. Original error: could not get source code\n",
+      "\u001b[2m\u001b[36m(pid=1366)\u001b[0m To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert\n",
+      "6/6 [==============================] - 0s 579us/step - loss: 15.9879 - mae: 2.7070\n",
+      "\u001b[2m\u001b[36m(pid=1366)\u001b[0m 2021-01-27 17:58:37.301439: W tensorflow/core/common_runtime/eager/context.cc:566] Unable to destroy server_ object, so releasing instead. Servers don't support clean shutdown.\n"
+     ]
+    }
+   ],
+   "source": [
+    "batch_size = 16\n",
+    "num_epochs = 80\n",
+    "# Generate a new estimator.\n",
+    "est = Estimator.from_keras(model_creator=build_model,\n",
+    "                           config={},\n",
+    "                           workers_per_node=1,\n",
+    "                           verbose=0)\n",
+    "# Train it on the entirety of the data.\n",
+    "stats = est.fit(train_data_creator, \n",
+    "                epochs=num_epochs,\n",
+    "                batch_size=batch_size,\n",
+    "                steps_per_epoch=600 // batch_size,\n",
```

Comment:
why here changes from 450 to 600?

Reply:
Because the size of training set changed from 3/4 of the set to the whole set.